src/components/landing/Footer.tsx

```diff
@@ -0,0 +1,86 @@
+import Image from 'next/image';
+import Link from 'next/link';
+import styled from 'styled-components';
+import MEDIA_QUERIES from '@/constants/MEDIAQUERIES';
+import Emil from '@/public/icon/email.svg';
+import Facebook from '@/public/icon/facebook.svg';
+import Insta from '@/public/icon/insta.svg';
+
+const S = {
+  Container: styled.footer`
+    display: flex;
+    align-items: center;
+    flex-direction: row;
+    justify-content: space-between;
+    height: 33.6rem;
+    padding: 12rem 14rem 0 14rem;
+    background-color: ${({ theme }) => theme.color.black_000000};
+
+    ${MEDIA_QUERIES.onTablet} {
+      flex-direction: row;
+      justify-content: space-between;
+      padding-left: 4rem;
+      padding-right: 4rem;
```

Comment:
[P3]
: 아래와 같이 써볼 수 있겠네요!
```suggestion
      padding: 0 4rem;
```